src/app-layer-htp-file.c

```diff
@@ -186,7 +186,7 @@ int HTPParseContentRange(bstr * rawvalue, HtpContentRange *range)
 
     if (data[pos] == '*') {
         // case with size only
-        if (len < pos + 1 || data[pos+1] != '/') {
+        if (len < pos + 2 || data[pos+1] != '/') {
```

Comment:
Suggest `<=`  with `pos + 1` instead (here, and at other change).

Reply:
ok